jwql/edb/engineering_database.py

```diff
@@ -45,7 +45,11 @@
 
 from collections import OrderedDict
 from datetime import datetime
+from enum import unique
```

Comment:
I see `numpy.unique` being used in the code, but I don't see `enum.unique` being used.  Could this be removed?